javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,97 @@
+// voice assistant 
+
+let userName= "";
+const todolist=[];
+
+function getReply(command){
+if (command==="Hello my name is Benjamin" && userName==null){
+    return `You are not a person`
+}
+    if (command ==="Hello my name is Benjamin" && userName===""){
+        userName="Benjamin";
+        return `Nice to meet you ${userName}`;
+
+    } if(command ==="Hello my name is Benjamin" && userName!=""){
+        userName="";
+        return`This name was already mentioned`;
```

Comment:
You should check which name is written and compare it with the userName, and then if those two are equal, return the sentence `This name was already mentioned`.
No need for using backticks here, check the article I posted previously.